external/fv3fit/fv3fit/_shared/config.py

```diff
@@ -71,17 +71,34 @@ class TrainingConfig:
     """
 
     model_type: str
-    input_variables: List[str]
```

Comment:
Also remove docstring lines for these, in addition to the `additional_variables`

Reply:
Thanks! Done.